Formula/pocketbase.rb

```diff
@@ -0,0 +1,19 @@
+class Pocketbase < Formula
+  desc "Open source backend for your next project in 1 file"
+  homepage "https://pocketbase.io/"
+  url "https://github.com/pocketbase/pocketbase/archive/refs/tags/v0.15.2.tar.gz"
+  sha256 "abc524d9a4e9235e6b2d9d274df1a9fd987f17d42427f3b085f777c064a29fd9"
+  license "MIT"
+
+  depends_on "go" => :build
+
+  def install
+    ENV["CGO_ENABLED"] = "0"
+
+    system "go", "build", *std_go_args(ldflags: "-s -w -X github.com/pocketbase/pocketbase.Version=#{version}"), "-o", bin/"pocketbase", "./examples/base"
+  end
+
+  test do
+    assert_match "pocketbase version #{version}", shell_output("#{bin}/pocketbase --version")
```

Comment:
We need a test that exercises the some of the functionality of the app. Version checks or usage checks (foo --version or foo --help) are not sufficient, as explained in the formula cookbook.

In most cases, a good test would involve running a simple test case: run #{bin}/foo input.txt.

- Then you can check that the output is as expected (with assert_equal or assert_match on the output of shell_output)
- You can also check that an output file was created, if that is expected: assert_predicate testpath/"output.txt", :exist?

Some advice for specific cases:

- If the formula is a library, compile and run some simple code that links against it. It could be taken from upstream's documentation / source examples.
- If the formula is for a GUI program, try to find some function that runs as command-line only, like a format conversion, reading or displaying a config file, etc.
- If the software cannot function without credentials, a test could be to try to connect with invalid credentials (or without credentials) and confirm that it fails as expected.
- Same if the software requires a virtual machine, docker instance, etc. to be running.

Reply:
Hello, thanks for your feedback!

I added some tests to check that the binary creates the default files and data directory for the application. Also, I have changed the installation command to the default.